src/main/java/org/synyx/urlaubsverwaltung/person/MailNotification.java

```diff
@@ -14,9 +15,12 @@ public enum MailNotification {
     NOTIFICATION_EMAIL_APPLICATION_MANAGEMENT_EDITED,
     NOTIFICATION_EMAIL_APPLICATION_MANAGEMENT_CONVERTED,
     NOTIFICATION_EMAIL_APPLICATION_MANAGEMENT_WAITING_REMINDER,
-
     NOTIFICATION_EMAIL_APPLICATION_MANAGEMENT_CANCELLATION_REQUESTED,
 
+    // colleagues
+    NOTIFICATION_EMAIL_COLLEAGUES,
```

Comment:
ich würde das hier in mehrere aufsplitten.

NOTIFICATION_EMAIL_COLLEAGUES_APPLIED
NOTIFICATION_EMAIL_COLLEAGUES_CANCELLATION